spec/index.bs

```diff
@@ -6157,6 +6159,7 @@ The completion of the fence and the value it completes with can be observed from
 <dl dfn-type=method dfn-for=GPUFence>
     : <dfn>getCompletedValue()</dfn>
     ::
+        Returns the fence's current completion value.
```

Comment:
The client side state here isn't defined yet, but we can be more specific here anyway, e.g: "Returns the largest signalled value that has propagated to the [=content timeline=]."